trl/trainer/utils.py

```diff
@@ -1781,6 +1781,30 @@ def selective_log_softmax(logits, index) -> torch.Tensor:
     return per_token_logps
 
 
+def rowise_entropy(logits, chunk_size: int = 1) -> torch.Tensor:
```

Comment:
```suggestion
def entropy_from_logits(logits, chunk_size: int = 1) -> torch.Tensor:
```

make sure to change test names as well